internal/core/data/v2/infrastructure/redis/client.go

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
It was decided in the Core WG meeting that all the redis implementation files should be in same location under internal/pkg/v2/infrastructure/redis similar to how it is V1.

Reply:
Updated the path to `internal/pkg/v2/infrastructure/redis`.